autoshop/autoshop/settings.py

```diff
@@ -23,9 +23,9 @@
 SECRET_KEY = 'i%&#7%nq24s#4xp+4b)w@$h)cy&nwotlz&pzb*s#kivh_6f#j6'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
This is the right choice. But for local development we need to run django in debug mode. Please create a `private.py` settings file and override this value there